site/gatsby-site/src/components/RelatedIncidents.js

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect, useState } from 'react';
+import { ListGroup } from 'react-bootstrap';
+import Loader from 'components/Loader';
+
+import config from '../../config';
+import { useMongo } from 'hooks/useMongo';
+
+const RelatedIncidents = ({ incident = {} }) => {
```

Comment:
New Issue (high priority): return related incidents under the following circumstances...

1) All incident reports with publication dates that are 2 weeks before or 2 weeks after the current publication date
2) All incident reports with dates that are 2 weeks before or 2 weeks after the current incident date

The newer `epoch_date_published` and `epoch_incident_date` fields should make this easier.

This is important because incident reports tend to land first with a single article then a blast of articles at a later date.